.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] There is a .gitignore file listing `test.md` file and `node_modules`. Lets include `node_modules` also.

Reply:
Thanks for the feedback. I have implemented it all